src/paper/tasks.py

```diff
@@ -995,6 +995,24 @@ def pull_biorxiv(page=0, retry=0):
         sentry.log_error(e)
 
 
+def _get_biorxiv_tweet_counts(url, doi):
+    try:
+        res = requests.get(
+            f"https://connect.biorxiv.org/eval_get1.php?url={url}&doi={doi}"
+        )
+        unescaped_json_res = res.text
+        # For some reason, the request data comes back with "\ufeff" and then also has an open paren ( and a closed paren )
+        escaped_json_res = json.loads(res.text.strip("\ufeff(").rstrip(res.text[-1]))
+        count = 0
+        for message in escaped_json_res["messages"]:
+            count += message["count_tweets"] + message["count_retweets"]
+        return count
+    except Exception as e:
+        print(e)
```

Comment:
Delete

Reply:
I'd like to keep it in here, otherwise during development these errors are swallowed up

Reply:
better to print it to the console IMO